src/main/java/teammates/common/util/Const.java

```diff
@@ -149,6 +149,11 @@ private Const() {
         public static final String ENTITY_TYPE = "entitytype";
 
         public static final String INTENT = "intent";
+
+        public static final String QUERY_LOGS_STARTTIME = "startTime";
+        public static final String QUERY_LOGS_ENDTIME = "endTime";
+        public static final String QUERY_LOGS_SEVERITIES = "severities";
+        public static final String NEXT_PAGE_TOKEN = "nextPageToken";
```

Comment:
All other items in `ParamsNames` are all-lowercase, so this should follow suit.

Reply:
Changed to lowercase.